### ISSUE TITLE
a11y contrast issues with disabled text

### ISSUE THREAD

**Issue by grigasp** [iTwin/stratakit#1143]
When rendering disabled text (`color: var(--stratakit-color-text-neutral-disabled)`) on default background color (`background-color: var(--stratakit-color-bg-page-base)`), we're getting a11y contrast warnings:

- Light theme: `Element has insufficient color contrast of 2.1 (foreground color: #acb1b7, background color: #fbfcfd, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1`

- Dark theme: `Element has insufficient color contrast of 2.52 (foreground color: #585f68, background color: #1f2023, font size: 9.0pt (12px), font weight: normal). Expected contrast ratio of 4.5:1`
